proposed/http-message.md

```diff
@@ -383,6 +383,20 @@ interface MessageInterface
     public function hasHeader($name);
 
     /**
+     * Retrieves a header by the given case-insensitive name as an array of
+     * strings.
+     *
+     * If the header does not appear in the message, this method MUST return an
+     * empty array.
+     *
+     * @param string $name Case-insensitive header field name.
+     * @return string[] An array of discrete string values for the given header.
```

Comment:
I'm not sure what you mean by discrete here. I think it muddies the language unless clarified. For example, getHeader() should not parse each header in order to separate out CSV. If a user calls `setHeader('foo', ['bar, baz', 'bam'])`, then calling `getHeader('foo')` should return `['bar, baz', 'bam']`.